### PR TITLE
[14.0][MIG] account_payment_multi_deduction

### PR DESCRIPTION
**Pull request by Saran440**
Odoo 14 change reigster payment object split to account.register.payment only
I refactor code to support v14 but functional is not change.

Cherry pick 1 commit from #361 

cc: @kittiu

### CODE REVIEW

File: account_payment_multi_deduction/readme/HISTORY.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+13.0.1.0.0 (2020-01-27)
```
Comment (kittiu) [inline account_payment_multi_deduction/readme/HISTORY.rst:1]: Please add history for migration.

**Comment by kittiu** [comment]
Functional tested and code review. LGTM.

**Review by kittiu — APPROVED** [review]

**Comment by kittiu** [comment]
I got following error, when I run test on command line for a new database. Can you check also.
<pre>
2020-12-02 04:46:36,582 163593 INFO 14.0-mig-account_payment_multi_deduction odoo.modules.loading: loading account_payment_multi_deduction/wizard/account_payment_register_views.xml 
2020-12-02 04:46:36,601 163593 INFO 14.0-mig-account_payment_multi_deduction odoo.modules.loading: Module account_payment_multi_deduction: loading demo 
2020-12-02 04:46:36,681 163593 INFO 14.0-mig-account_payment_multi_deduction odoo.addons.account_payment_multi_deduction.tests.test_payment_multi_deduction: Starting TestPaymentMultiDeduction.test_one_invoice_payment ... 
2020-12-02 04:46:36,978 163593 INFO 14.0-mig-account_payment_multi_deduction odoo.addons.account_payment_multi_deduction.tests.test_payment_multi_deduction: ====================================================================== 
2020-12-02 04:46:36,978 163593 ERROR 14.0-mig-account_payment_multi_deduction odoo.addons.account_payment_multi_deduction.tests.test_payment_multi_deduction: ERROR: TestPaymentMultiDeduction.test_one_invoice_payment
Traceback (most recent call last):
  File "/home/kittiu/PycharmProjects/account-payment/account_payment_multi_deduction/tests/test_payment_multi_deduction.py", line 121, in test_one_invoice_payment
    payment = self.payment_model.browse(payment_dict.get("res_id", False))
AttributeError: 'bool' object has no attribute 'get'
 
2020-12-02 04:46:36,978 163593 INFO 14.0-mig-account_payment_multi_deduction odoo.addons.account_payment_multi_deduction.tests.test_payment_multi_deduction: Starting TestPaymentMultiDeduction.test_one_invoice_payment_foreign_currency ... 
2020-12-02 04:46:37,103 163593 INFO ? odoo.http: HTTP Configuring static files 
2020-12-02 04:46:37,284 163593 INFO 14.0-mig-account_payment_multi_deduction odoo.addons.account_payment_multi_deduction.tests.test_payment_multi_deduction: ====================================================================== 
2020-12-02 04:46:37,285 163593 ERROR 14.0-mig-account_payment_multi_deduction odoo.addons.account_payment_multi_deduction.tests.test_payment_multi_deduction: ERROR: TestPaymentMultiDeduction.test_one_invoice_payment_foreign_currency
Traceback (most recent call last):
  File "/home/kittiu/PycharmProjects/account-payment/account_payment_multi_deduction/tests/test_payment_multi_deduction.py", line 176, in test_one_invoice_payment_foreign_currency
    payment = self.payment_model.browse(payment_dict.get("res_id", False))
AttributeError: 'bool' object has no attribute 'get'
 
2020-12-02 04:46:37,285 163593 INFO 14.0-mig-account_payment_multi_deduction odoo.addons.account_payment_multi_deduction.tests.test_payment_multi_deduction: Starting TestPaymentMultiDeduction.test_one_invoice_payment_with_keep_open ... 
2020-12-02 04:46:37,502 163593 INFO 14.0-mig-account_payment_multi_deduction odoo.addons.account_payment_multi_deduction.tests.test_payment_multi_deduction: ====================================================================== 
2020-12-02 04:46:37,502 163593 ERROR 14.0-mig-account_payment_multi_deduction odoo.addons.account_payment_multi_deduction.tests.test_payment_multi_deduction: ERROR: TestPaymentMultiDeduction.test_one_invoice_payment_with_keep_open
Traceback (most recent call last):
  File "/home/kittiu/PycharmProjects/account-payment/account_payment_multi_deduction/tests/test_payment_multi_deduction.py", line 243, in test_one_invoice_payment_with_keep_open
    payment = self.payment_model.browse(payment_dict.get("res_id", False))
AttributeError: 'bool' object has no attribute 'get'

</pre>

**Comment by kittiu** [comment]
Ping @Saran440

**Comment by Saran440** [comment]
@kittiu  Fixed, Thank you for remined me

**Pull request merged**